sol-3.sql

```diff
@@ -0,0 +1,95 @@
+-- những cặp student-professor có dạy học nhau và số lớp mà họ có liên quan
+select p.prof_id, e.stud_id, count(*) as 'no_of_class'
+from professor as p join class as c on p.prof_id = c.prof_id
+join enroll as e on e.class_id
+group by p.prof_id, e.stud_id;
+
+-- những course (distinct) mà 1 professor cụ thể đang dạy
+select distinct c.course_id, c.course_name, p.prof_id, p.prof_fname, p.prof_lname
+from professor as p join class as cl on p.prof_id = cl.prof_id
+join course as c on cl.course_id = c.course_id;
```

Comment:
this question meaning that, we already have prof_id, and find all distinct course that he teach